README.md

```
@@ -93,12 +131,27 @@ time, so neither can drift from the schema.
+generated rather than committed:
+
+```sh
+node docs/generate_reference.mjs
+pip install -r docs/requirements.txt
+mkdocs serve
+```
+
 ## License
 
-The Integra Framework is licensed under the GNU GPL version 2. For further details see [here](libIntegra/LICENSE.txt)
+The Integra Framework is licensed under the GNU GPL version 2 or, at your option, any later version. For further details see [here](libIntegra/LICENSE.txt)
```

markdownlint-cli2 (0.23.2)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)